### PR TITLE
Coarse drafts of CONTRIBUTING and README

### PR DESCRIPTION
**Pull request by aprokop**

### CODE REVIEW

**Comment by aprokop** [comment]
Lets merge this in some form, so that our front page looks semi-decent.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Contributing
+------------
+
+Contributing to ArborX is easy. Just send us a [pull request](https://help.github.com/articles/using-pull-requests/).
+When you send your request, make `master` the destination branch on the
+[ArborX repository](https://github.com/ArborX/ArborX).
+
+Your pull request must pass ArborX's tests. We enforce these guidelines with Jenkins CI.
```
Comment (dalg24) [inline CONTRIBUTING.md]: What guidelines?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+ArborX
+======
+
+[![Build Status](https://cloud.cees.ornl.gov/jenkins-ci/buildStatus/icon?job=ArborX-continuous)](https://cloud.cees.ornl.gov/jenkins-ci/job/ArborX-continuous/)
+[![Documentation Status](http://readthedocs.org/projects/arborx/badge/?version=latest)](http://arborx.readthedocs.io/en/latest/?badge=latest)
+[![codecov](https://codecov.io/gh/ArborX/ArborX/branch/master/graph/badge.svg)](https://codecov.io/gh/ArborX/ArborX)
```
Comment (dalg24) [inline README.md]: No badges for now (Jenkins link does not exist, doc has nothing, coverage is not setup)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+ArborX is an open-source library designed to provide performance portable
+algorithms for geometric search, similarly to nanoflann and Boost Geometry.
+
+Documentation
+-------------
+
+The documentation and the installation instructions can be found
+[here](http://arborx.readthedocs.org).
```
Comment (dalg24) [inline README.md]: No doc anywhere

**Comment by aprokop** [comment]
Rebased on `master` to be able to run the tests.

**Pull request merged**